setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I didn't want to imply you had to copy the whole windows api code pack :) Just the class which produces the folder browser, that's all. I don't think that's a lot of bloat. The thing with adding a nuget package is that the dlls have to be shipped with the tool, or you have to ilmerge them together into the exe. It's currently a single exe that's shipped, I think with the dlls added to that you need extra instructions how to deal with the zip file etc. But I'm not the person who has the end-say over this, so if crosire is fine with it, I don't mind, I was merely pointing into the direction where a solution might be found ;)